### DISCUSSION TITLE
How evaluate own test data

### DISCUSSION THREAD

**Discussion by VladHardAndSoftWare** [RUCAIBox/RecBole#1611] [Q&A] [answered]
I want evaluate data on the current day. Instead of using the standard division. To do this, I used:
`data_sampler_train = get_dataloader(config, 'train')` 
`data_train = TrainDataLoader(config, dataset_train, data_sampler_train)` <br />and 
`data_sampler_test = get_dataloader(config_test, 'evaluation')` 
`data_test = FullSortEvalDataLoader(config_test, dataset_test, data_sampler_test)`

but I got the following error:
![image](https://user-images.githubusercontent.com/42782084/210109792-f33c2f18-246e-4104-ac88-72c64dbbbe9e.png)

Can you tell me if it is possible to predict my own test data?

**Comment by Ethan-TZ** [comment, marked as answer]
@VladHardAndSoftWare Thanks your attention to RecBole!
This error may have been caused by an incorrect use of **dataloader**.
For predicting your current day data by a pre-trained model, you can try to utilize our provided [case study](https://recbole.io/docs/user_guide/usage/case_study.html#case-study) module.
I hope my answer could help you!
